addon.js

```diff
@@ -1,4 +1,7 @@
 var inlineTemplateCompiler = require('./index');
+var path = require('path');
+// for ember-cli always use the bower version of the ember template compiler
+var compiler = require(path.join(process.cwd(),'./bower_components/ember/ember-template-compiler.js'));
```

Comment:
This is not the guaranteed path at file load time.  Can you move this require into the `init` of the addon and use `this.project.root()` to get the project path?